src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -9,6 +9,18 @@ title: "Managing Nodes"
 This topic describes how to manage nodes on kURL clusters.
 It includes procedures for how to safely reset, reboot, and remove nodes when performing maintenance tasks.
 
+See the following sections:
+
+* [ECKO Add-On Prerequisite](#ecko-add-on-prerequisite)
+* [Reset a Node](#reset-a-node)
+* [Reboot a Node](#reboot-a-node)
+* [Remove a Node from Rook Ceph Clusters](#remove-a-node-from-rook-ceph-clusters)
+   * [Rook Ceph and etcd Node Removal Requirements](#rook-ceph-and-etcd-node-removal-requirements)
+   * [Rook Ceph Cluster Prerequisites](#rook-ceph-cluster-prerequisites)
+   * [(Recommended) Manually Rebalance Ceph and Remove a Node](#recommended-manually-rebalance-ceph-and-remove-a-node)
+   * [Remove Nodes with ECKO](#remove-nodes-with-ecko)
+* [Troubleshoot Node Removal](#troubleshoot-node-removal)
```

Comment:
What about Known issues with Node Removal ?

Reply:
I'm going to leave it as Troubleshoot, since we are providing troubleshooting steps.